doc/src/devdocs/stdio.md

```diff
@@ -2,37 +2,37 @@
 
 ## Libuv wrappers for stdio
 
-`julia.h` defines [libuv](http://docs.libuv.org) wrappers for the `stdio.h` streams:
+julia.h defines [libuv](http://docs.libuv.org) wrappers for the stdio.h streams:
```

Comment:
Okay, well either way it's inconsistent. I'm inclined to just merge this then make everything consistent in another PR.